docs/src/prototypes/chatMessages/ControlMessages/index.tsx

```diff
@@ -0,0 +1,198 @@
+import overridenTreeTitleBehavior from 'docs/src/prototypes/chatMessages/ControlMessages/overridenTreeTitleBehavior'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: {
+    color: 'green',
+    icon: 'check',
+  },
+}
+
+const treeItems = [
+  render =>
+    render({}, (Component, props) => {
+      const createdRef = React.createRef<HTMLButtonElement>()
+      return (
+        <Component
+          {...props}
+          key="11"
+          accessibility={overridenTreeItemBehavior}
+          title={render =>
+            render(
+              {
+                onKeyDown: e => {
+                  if (keyboardKey.getCode(e) === keyboardKey.Escape) {
+                    e.stopPropagation()
+                    createdRef.current.focus()
+                  }
+                },
+                as: 'div',
+                accessibility: overridenTreeTitleBehavior,
+                content: (
+                  <div>
+                    <a href="/">John Doe1</a> added <a href="/">Jane Doe1</a> to the conversation
+                  </div>
+                ),
+              },
+              (Component, props) => (
+                <Ref innerRef={createdRef}>
+                  <Component {...props} />
+                </Ref>
+              ),
+            )
+          }
+        />
+      )
+    }),
+  render =>
+    render({}, (Component, props) => {
+      const createdRef = React.createRef<HTMLButtonElement>()
+      return (
+        <Component
+          {...props}
+          accessibility={overridenTreeItemBehavior}
+          key="21"
+          title={render =>
+            render(
+              {
+                onKeyDown: e => {
+                  if (keyboardKey.getCode(e) === keyboardKey.Escape) {
+                    e.stopPropagation()
+                    createdRef.current.focus()
+                  }
+                },
+                as: 'div',
+                accessibility: overridenTreeTitleBehavior,
+                content: (
+                  <div>
+                    <a href="/">John Doe2</a> added <a href="/">Jane Doe2</a> to the conversation
+                  </div>
+                ),
+              },
+              (Component, props) => (
+                <Ref innerRef={createdRef}>
+                  <Component {...props} />
+                </Ref>
+              ),
+            )
+          }
+        />
+      )
+    }),
+]
+
+const ChatExample = () => {
+  const [open, setOpen] = React.useState(false)
+
+  const items: ShorthandCollection<ChatItemProps> = [
+    {
+      message: {
+        onKeyDown: e => {
+          const eventCode = keyboardKey.getCode(e)
+          if (eventCode === keyboardKey.Enter || eventCode === keyboardKey.Spacebar) {
+            setOpen(true)
+          }
+          if (eventCode === keyboardKey.Escape) {
+            setOpen(false)
+          }
+        },
+        content: (
+          <Tree
```

Comment:
Looking at the chat example, it was not clear to me that this is actually an expansible tree.

Reply:
The styling are not implemented for this prototype, we are focusing only on the behavior and whether everything works as expected